site/_data/exampleapps.yaml

```diff
@@ -219,3 +219,7 @@
   name: Twitter login
   description: Node/express app which uses Twitter for authentication
   language: javascript
+- url: https://github.com/FusionAuth/fusionauth-example-react-sdk
```

Comment:
Is it confusing that that the React SDK example app shows up under a React heading, and the React authorization code grant example shows up under a Javascript heading?

Reply:
Those don't go under the react SDK because they aren't using the SDK. But I'll mark them so in the description. We want to deprecate these applications, so it probably makes sense to archive them and point to the react SDK example app.